test/dsl/dsl_advanced_model_construction.jl

```diff
@@ -131,7 +131,7 @@ let
     end
     # Line number nodes aren't ignored so have to be manually removed
     Base.remove_linenums!(ex)
-    @test eval(Catalyst.make_reaction_system(ex)) isa ReactionSystem
+    @test eval(Catalyst.make_reaction_system(ex, QuoteNode(:name))) isa ReactionSystem
```

Comment:
adapted to the new `make_reaction_system` function (which won't generate names for the user)